### PR TITLE
Release 0.6

### PR DESCRIPTION
**Pull request by tkdchen**

### CODE REVIEW

File: message_tagging_service/tagging_service.py [hunk 1]
```diff
@@ -344,7 +344,6 @@ def tag_build(nvr, dest_tags, koji_session):
             else:
                 task_id = koji_session.tagBuild(tag, nvr)
         except Exception as e:
-            logger.exception('Failed to tag %s in %s', nvr, tag)
```
Comment (lcarva) [inline message_tagging_service/tagging_service.py]: Given that we're catching any exception, do you think it's beneficial to still log the traceback?

Comment (tkdchen) [reply message_tagging_service/tagging_service.py]: This try-except only wraps the tagBuild call. What I can see from the log output, the error message is unnecessarily shown repeatedly, for example,

```
  File "/usr/lib/python3.7/site-packages/koji/__init__.py", line 2546, in _callMethod
    raise err
koji.TagError: Package e2e-module-test-devel not in list for release-e2e-test-1.0-rhel-8-modules-candidate
2019-09-26 10:33:04,977- message_tagging_service.tagging_service - WARNING - None of tag(s) ['release-e2e-test-1.0-rhel-8-modules-candidate'] is applied to build e2e-module-test-devel-1.0-8020020190926101711.483c918a successfully.
2019-09-26 10:33:04,977- message_tagging_service.tagging_service - WARNING - Tag release-e2e-test-1.0-rhel-8-modules-candidate. Failure reason: Package e2e-module-test-devel not in list for release-e2e-test-1.0-rhel-8-modules-candidate
```

In this context, I think it doesn't make too much sense to log tracking for tracking the failure reason from tagBuild.

Comment (lcarva) [reply message_tagging_service/tagging_service.py]: Well sure, but `koji.TagError` is not the only error that could be raised. For instance, if a tag does not exist, you get a `koji.GenericError`. That's just an example. What if there's a kerberos error? Or requests issue? Maybe we can restructure the `try/except` block to catch specific errors, and have a generic one for any other errors:

```python
try:
  tagBuild
except koji.TagError:
  # Display message with error details
except Exception:
  logger.exception(...)
  
```

Comment (tkdchen) [reply message_tagging_service/tagging_service.py]: Reverted the revert.

**Pull request merged**